AEXS/aex-141.md

```diff
@@ -13,21 +13,23 @@ Created: 2021-09-11
 
 ## Abstract
 
-A standard implementation of non-fungible tokens for the Aeternity ecosystem. The design goal of the primary interface is to be as compatible with ERC-721 as possible, so that anyone who can work with ERC-721 can work with this interface. However, where Sophia offers a better way, performance and efficiency should prevail over compatibility.
+A standard implementation of non-fungible tokens for the æternity ecosystem. The design goal of the primary interface is to be as compatible with ERC-721 as possible, so that anyone who can work with ERC-721 can work with this interface. However, where Sophia offers a better way, performance and efficiency should prevail over compatibility.
```

Comment:
is this still true, maybe we should add a "how we differentiate from ERC 721" section

Reply:
updated: https://github.com/aeternity/AEXs/commit/2820512fba310a10bb3c15bce8b40c9eda016498